docs/pages/blog/tenerife-retreat.js

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import TopLayoutBlog from 'docs/src/modules/components/TopLayoutBlog';
+import { docs } from './tenerife-retreat.md?@mui/markdown';
```

Comment:
Maybe it would help with the year, to give some context

```suggestion
import { docs } from './2022-tenerife-retreat.md?@mui/markdown';
```